.circleci/config.yml

```diff
@@ -580,11 +582,11 @@ workflows:
       - nightly-owasp-scan
     triggers:
       - schedule:
-          cron: "0 0 * * *"
+          cron: "10,40 * * * *"
```

Comment:
this is a temporary change to cron job frequency for testing. this runs every hour at 10 after and 40 after on `feat/1534-updating-to-latest-zaproxy` branch. this should be changed back to default before merge.